src/SlmLocale/Strategy/Factory/UriPathStrategyFactory.php

```diff
@@ -65,7 +65,7 @@ class UriPathStrategyFactory implements FactoryInterface
      */
     public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
-        $strategy = new UriPathStrategy($container->get('router'));
+        $strategy = new UriPathStrategy($container->getServiceLocator()->get('router'));
```

Comment:
this will not work on ServiceManager v3 instead add this to createService() method and then change 
```
$strategy->setServiceLocator($container);
```
to:
```
$strategy->setServiceLocator($container->get(\SlmLocale\Strategy\StrategyPluginManager::class));
```

Reply:
Hey @svycka , thanks for the help, but I don't really feel comfortable making the final code change since I don't understand the v3 ServiceManager (I'm also no expert with v2). 
This pull request was more just to point out where the problem is and my little workaround. 

I'd appreciate it if you could make the change. Thanks for the quick responses!